src/engine/Frame.java

```diff
@@ -32,7 +32,7 @@ public class Frame extends JFrame {
 	 */
 	public Frame(final int width, final int height) {
 		setSize(width, height);
-		setResizable(false);
+		setResizable(true);
```

Comment:
setResizable을 true로 바꾼다면 다른 모든 화면에서도 사용자가 직접 화면을 조정할수 있기 때문에 변경해야 할 것 같습니다.

==========================================

If you change setResizable to true, it seems necessary to change it because the user can directly adjust the screen on all other screens.


<img width="898" alt="스크린샷 2024-10-15 오후 12 13 09" src="https://github.com/user-attachments/assets/23b4bd4e-1678-4120-b296-3df9d787bf7b">

Reply:
Okay, I'll leave resizable true for a while, only when I need it.